app/services/people/importer.rb

```diff
@@ -15,9 +15,10 @@ def initialize(nomis_attributes)
     end
 
     def call
-      person = Person.find_or_create_by!(nomis_prison_number: nomis_attributes[:prison_number])
+      person = Person.find_or_initialize_by(nomis_prison_number: nomis_attributes[:prison_number])
       profile = person.latest_profile || person.profiles.build
-      profile.update(attributes)
+      profile.update_attributes(attributes)
```

Comment:
It looks like `update_attributes` is just an alias of `update` (https://apidock.com/rails/ActiveRecord/Persistence/update_attributes). Is there a reason we're switching?

Reply:
Yes this should be assign_attributes...